reconcile/slack_base.py

```diff
@@ -2,12 +2,12 @@
 
 from reconcile import queries
 from reconcile.utils.secret_reader import SecretReader
-
 from reconcile.utils.slack_api import SlackApi, SlackApiConfig
+from reconcile.utils.slack_api import SupportsClientConfig
 
 
 def slackapi_from_queries(
-    integration_name: str, init_usergroups: Optional[bool] = True
+    integration_name: str, init_usergroups: bool = True
```

Comment:
Before the change, could `init_usergroups` have been None?  Wonder what the actual diff here is.

Reply:
No, is always set to either `True`/`False` or not given.